docs/CNAME

```diff
@@ -0,0 +1 @@
+smlhelp-demo.camdar.io
```

Comment:
Should probably chance (or potentially remove the file altogether)?

Reply:
Is this file required at all?

Reply:
It isn't, and I don't know how it got committed (probably due to my setting up the custom domain on my end)